### ISSUE TITLE
resin env not supported on multicontainer apps

### ISSUE THREAD

**Issue by jgentes** [balena-io/balena-cli#855]
I get this error when I try to set an environment variable on my multi-container app: `ResinRequestError: Request error: Bad Request`

Here's my code:
```
resin login --token $RESIN_AUTH
resin env add RELEASE_SHA 'test' --app 'myApp'
```

I see the same error when I try to perform the operation using the node resin-sdk. Any thoughts?

This is the latest resin-cli and I'm doing this from a Heroku server (x64 linux)

<img src="https://frontapp.com/assets/img/icons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_d8lv)

**Comment by pimterry** [comment]
This'll be caused by the underlying SDK not having support for this yet (https://github.com/resin-io/resin-sdk/issues/488). Once the SDK has that, and the CLI is updated, this should start working straight away.

**Comment by pimterry** [comment]
Just to update on this: the CLI should now work with multicontainer environment and config variables, and you shouldn't see this error any more. We don't yet have support for configuring per-service variables in the CLI (only application-wide environment variables), but we will be working on adding that in future.

**Comment by magiconair** [comment]
@pimterry when will you add support for setting service variables and why don't you add this at the same time when you add device variable support?

**Comment by pdcastro** [comment]
@magiconair, thanks for raising this matter. I have created a new more specific issue:
[Add support for setting per-service variables (multi-container / multi-service)](https://github.com/balena-io/balena-cli/issues/1153)

I cannot provide an ETA but I acknowledge that it is an important high-priority feature request.

> _[...] why don't you add this at the same time when you add device variable support?_

To confirm, setting device variables is already supported (`--device` option). I think that support for that feature was added before balena had support for multi-container / multi-service apps (not completely sure though).
